docker/cluster.py

```diff
@@ -0,0 +1,200 @@
+import argparse
+import json
+import os
+import shutil
+import subprocess
+
+import docker
+from docker.errors import NotFound
+from keywords.exceptions import DockerError
+from keywords.utils import log_info
+
+
+def setup_port_mapping(current_port):
+    """ Expose ports to host for local development """
+
+    port_map = {}
+    ports_to_map = ['8091/tcp', '4984/tcp', '4985/tcp']
+    for port in ports_to_map:
+        log_info('Setting up port binding: {} -> localhost:{}'.format(port, current_port))
+        port_map[port] = current_port
+        current_port += 1
+
+    return port_map, current_port
+
+
+def create_and_bind_tmp_dirs(container_name, volume_map):
+    """ Create directories for binding the log directories for the various services.
+    This provides quicker inspection / collection of the logs during devlopment """
+
+    tmp_dir_sg = '/tmp/{}-sg'.format(container_name)
+    tmp_dir_cbs = '/tmp/{}-cbs'.format(container_name)
```

Comment:
We are not really sure what will be installed so it sets up bindings for any possible service.